front/src/Phaser/Entity/PlayerTextures.ts

```diff
@@ -10,440 +10,94 @@ export interface BodyResourceDescriptionInterface {
+    public static LAYERS: BodyResourceDescriptionListInterface[];
 
-export const HATS_RESOURCES: BodyResourceDescriptionListInterface = {
-    hats_1: { name: "hats_1", img: "resources/customisation/character_hats/character_hats1.png" },
-    hats_2: { name: "hats_2", img: "resources/customisation/character_hats/character_hats2.png" },
-    hats_3: { name: "hats_3", img: "resources/customisation/character_hats/character_hats3.png" },
-    hats_4: { name: "hats_4", img: "resources/customisation/character_hats/character_hats4.png" },
-    hats_5: { name: "hats_5", img: "resources/customisation/character_hats/character_hats5.png" },
-    hats_6: { name: "hats_6", img: "resources/customisation/character_hats/character_hats6.png" },
-    hats_7: { name: "hats_7", img: "resources/customisation/character_hats/character_hats7.png" },
-    hats_8: { name: "hats_8", img: "resources/customisation/character_hats/character_hats8.png" },
-    hats_9: { name: "hats_9", img: "resources/customisation/character_hats/character_hats9.png" },
-    hats_10: { name: "hats_10", img: "resources/customisation/character_hats/character_hats10.png" },
-    hats_11: { name: "hats_11", img: "resources/customisation/character_hats/character_hats11.png" },
-    hats_12: { name: "hats_12", img: "resources/customisation/character_hats/character_hats12.png" },
-    hats_13: { name: "hats_13", img: "resources/customisation/character_hats/character_hats13.png" },
-    hats_14: { name: "hats_14", img: "resources/customisation/character_hats/character_hats14.png" },
-    hats_15: { name: "hats_15", img: "resources/customisation/character_hats/character_hats15.png" },
-    hats_16: { name: "hats_16", img: "resources/customisation/character_hats/character_hats16.png" },
-    hats_17: { name: "hats_17", img: "resources/customisation/character_hats/character_hats17.png" },
-    hats_18: { name: "hats_18", img: "resources/customisation/character_hats/character_hats18.png" },
-    hats_19: { name: "hats_19", img: "resources/customisation/character_hats/character_hats19.png" },
-    hats_20: { name: "hats_20", img: "resources/customisation/character_hats/character_hats20.png" },
-    hats_21: { name: "hats_21", img: "resources/customisation/character_hats/character_hats21.png" },
-    hats_22: { name: "hats_22", img: "resources/customisation/character_hats/character_hats22.png" },
-    hats_23: { name: "hats_23", img: "resources/customisation/character_hats/character_hats23.png" },
-    hats_24: { name: "hats_24", img: "resources/customisation/character_hats/character_hats24.png" },
-    hats_25: { name: "hats_25", img: "resources/customisation/character_hats/character_hats25.png" },
-    hats_26: { name: "hats_26", img: "resources/customisation/character_hats/character_hats26.png" },
-    tinfoil_hat1: { name: "tinfoil_hat1", img: "resources/customisation/character_hats/tinfoil_hat1.png" },
-};
+    public static loadPlayerTexturesMetadata(url: string): Promise<boolean> {
```

Comment:
I don't think the "url" parameter should be passed here. The "/woka-list" URL can be hardcoded in the method body.